### PR TITLE
Ingot separation

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Recipes to dismantle ingots"

#### Purpose of change
Allows disassembly of 5L, 10L, and the Exodii's gigantic ingots.

#### Describe the solution
This was being worked on by someone else (who is no longer part of the project) so I didn't add them when I created the steel ingots. This picks up where they left off.

#### Describe alternatives you've considered
None

#### Testing
WIP

### CODE REVIEW

**Comment by Drew4484** [comment]
> I was considering giving these "charges" and then using a recipe to craft a chunk of their respective materials with metal sawing and using 1 charge of their "ammo".  It would require making the individual ingots weigh nothing and giving them a fake ammo type with no unload, and making them no reload, then weighting the ammo appropriately, but it would probably allow for more gradual disassembly of the ingots. Only issue I can think of is that it will give you "this X is too large and bulky to disassemble properly" so it could instead be activated with a metal sawing activity.

Not a bad idea but I think charges are on their way out, right?

File: data/json/uncraft/generic.json [hunk 1]
```diff
@@ -52,6 +52,222 @@
     "qualities": [ { "id": "SAW_M", "level": 2 } ],
     "components": [ [ [ "scrap", 5 ] ] ]
   },
+  {
+    "result": "exodii_ingot_tin",
+    "type": "uncraft",
+    "activity_level": "MODERATE_EXERCISE",
+    "time": "16 h",
+    "tools": [ [ [ "angle_grinder", 500 ] ] ],
```
Comment (phoeo) [inline data/json/uncraft/generic.json:60]: Why use a specific tool, if there are multiple tools with grinding quality?

Comment (Drew4484) [reply data/json/uncraft/generic.json:60]: The largest ingots are nearly a meter across. I'm all for adding other tools but doing it with hand tools isn't practical.

Comment (phoeo) [reply data/json/uncraft/generic.json:60]: Isn't angle grinder a hand tool, too?

What if stronger tools are added in the future? For example, in light industry you can find arc furnaces, and mines have unspecified heavy machinery, which may eventually include industrial grinders.

Comment (Drew4484) [reply data/json/uncraft/generic.json:60]: "Hand tools" in the sense of an unpowered tool, using only the user's strength and any mechanical advantage. We may need to revisit qualities like metal sawing but I'd like to hold off on that until we work out a way to improve craft times based on that tool quality.

**Pull request merged**